.github/workflows/build.yml

```diff
@@ -1,19 +1,29 @@
-name: Test & Build
-
+name: build
 on:
+  release:
+    types: [created]
   push:
-    branches: '*'
-  pull_request:
-    branches: '*'
-
+    branches:
+      - '**'
 jobs:
-  test-build:
-    name: Run tests and build project
+  build:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
```

Comment:
I believe this isn't published to the GH package registry, no? Only npm.
In that case we probably don't need to add `packages: write`?

Reply:
Yeah which I suppose this will never do so fair to remove it. Will update.

Reply:
8f2f01b4fc4bb096f39dd2f88a755bf3f096f319